lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -744,6 +765,8 @@ namespace Js
             return FALSE;
         }
```

Comment:
This is inconsistent compared to other methods. Can we move this before `handler == nullptr` check?